test/Peddler.Tests/DateTimeGeneratorTests.cs

```diff
@@ -100,91 +104,383 @@ public class DateTimeGeneratorTests : IntegralGeneratorTests<DateTime> {
         [Theory]
         [MemberData(nameof(Constructor_WithRange_MismatchedKinds_Data))]
         public void Constructor_WithRange_MismatchedKinds(DateTime low, DateTime high) {
-            Assert.Throws<ArgumentException>(
+
+            // Act
+
+            var exception = Record.Exception(
                 () => new DateTimeGenerator(low, high)
             );
+
+            // Assert
+
+            Assert.IsType<ArgumentException>(exception);
+        }
+
+        public static IEnumerable<object[]> KindsMemberData { get; } =
```

Comment:
Should this be named `UnitsMemberData`? I don't understand the name `KindsMemberData`

Reply:
Yes it should. I took this opportunity to clean up various member data and test names throughout the DateTime-based generator test code.